stable/weave-cloud/Chart.yaml

```diff
@@ -0,0 +1,11 @@
+name: weave-cloud
+version: 0.1.0
+description: |
+    Weave Cloud Agents
+
+    Weave Cloud is a add-on to Kubernetes which provides Continuous Delivery, along with hosted Prometheus Monitoring and a visual dashboard for exploring & debugging microservices
+home: https://weave.works
+maintainers:
+  - name: Ilya Dmitrichenko
+    email: ilya@weave.works
+engine: gotpl
```

Comment:
> @migmartri can you comment on what kind of images are supported on the Monocular/KubeApps side?

I believe that any of the HTML supported formats listed here https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#Supported_image_formats